Snakefile

```diff
@@ -414,6 +415,30 @@ rule trinity_stats:
     {TRINITY_HOME}/util/misc/plot_ExN50_statistic.Rscript {output.exN50} &>> {log}
     """
 
+rule busco:
+  """
+  Runs BUSCO to assess the completeness of the transcriptome.
+  """
+  input:
+    transcriptome="transcriptome.fasta"
+  output:
+    directory("busco")
```

Comment:
Maybe you should specify the key output files of busco here. Otherwise transXpress will not monitor those files and may consider this job as completed even if the busco run crashes (even a crashed run will probably produce the busco directory).

Reply:
Good point, I will change this.